client/src/app/pages/applications/components/application-review-status/application-review-status.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Application } from "@app/api/models";
+import { IconedStatus, IconedStatusPreset } from "@app/components/IconedStatus";
+import { Spinner } from "@patternfly/react-core";
+import { EmptyTextMessage } from "@app/components/EmptyTextMessage";
+import { useTranslation } from "react-i18next";
+import { useFetchArchetypes } from "@app/queries/archetypes";
+
+export interface ApplicationReviewStatusProps {
+  application: Application;
+  isLoading?: boolean;
+}
+
+export const ApplicationReviewStatus: React.FC<
+  ApplicationReviewStatusProps
+> = ({ application }) => {
+  const { t } = useTranslation();
+
+  const { archetypes, isFetching } = useFetchArchetypes();
+  const isAppReviewed = !!application.review;
```

Comment:
Is the review one for the application or does it include the archetype review?  This column may have the same issue as the assessment column...

Reply:
The review only includes the app level review. There is no review bool, this is just the actual review object on the application.